cmd/rancherd/go.sum

```diff
@@ -732,6 +732,8 @@ github.com/rancher/rke2 v1.18.13-0.20201116190518-b63f92a2b66e h1:0f5DPgfBH8+Y1m
 github.com/rancher/rke2 v1.18.13-0.20201116190518-b63f92a2b66e/go.mod h1:hQ5acWquDxv8rc7Nv1gXYEaZ2at4D6OwHdvlH7TxQLQ=
 github.com/rancher/rke2 v1.18.13-0.20201207193710-1d6e826b9fdf h1:YjHm/U2dMEUtYFCrLzto42IcsjVONfXoF2a7pcttIx4=
 github.com/rancher/rke2 v1.18.13-0.20201207193710-1d6e826b9fdf/go.mod h1:hQ5acWquDxv8rc7Nv1gXYEaZ2at4D6OwHdvlH7TxQLQ=
```

Comment:
does tidy clean up these old versions ?

Reply:
I don't think so. Unless you deleted go.sum and regenerate it

Reply:
I ran `go mod tidy` looks like it did get rid of the old version - pr ready for re-review